### ISSUE TITLE
Holybro X500 airframe doesn't set up motor outputs

### ISSUE THREAD

**Issue by DonLakeFlyer** [PX4/PX4-Autopilot#21347]
You end having to go into the Actuators page and do it yourself. And of course if you are a hardware moron like me you get it wrong and can't quite figure out why. This is a fully known dev kit. Hence the motors and motor output setup is a known thing as well. These parameters should all be set in the airframe for you.

**Comment by bkueng** [comment]
You're right, is this how it's assembled? https://github.com/PX4/PX4-Autopilot/pull/21349
